internal/provider/resource_aci_relation_to_domain_test.go

```diff
@@ -26,12 +26,8 @@ func TestAccResourceFvRsDomAttWithFvAEPg(t *testing.T) {
 					resource.TestCheckResourceAttr("aci_relation_to_domain.allow_test_2", "target_dn", "uni/vmmp-VMware/dom-domain_1"),
 					resource.TestCheckResourceAttr("aci_relation_to_domain.allow_test", "annotation", "orchestrator:terraform"),
 					resource.TestCheckResourceAttr("aci_relation_to_domain.allow_test_2", "annotation", "orchestrator:terraform"),
-					resource.TestCheckResourceAttr("aci_relation_to_domain.allow_test", "binding_type", "none"),
-					resource.TestCheckResourceAttr("aci_relation_to_domain.allow_test_2", "binding_type", "none"),
 					resource.TestCheckResourceAttr("aci_relation_to_domain.allow_test", "class_preference", "encap"),
 					resource.TestCheckResourceAttr("aci_relation_to_domain.allow_test_2", "class_preference", "encap"),
-					resource.TestCheckResourceAttr("aci_relation_to_domain.allow_test", "custom_epg_name", ""),
-					resource.TestCheckResourceAttr("aci_relation_to_domain.allow_test_2", "custom_epg_name", ""),
```

Comment:
Why are we removing these?

Reply:
Introduced version_mismatch in default which takes care of this.